### PR TITLE
43 GHSA-sync'ed files - minor edits

### PR DESCRIPTION
**Pull request by jasnow**
This files were modified by running the "sync_github_advisories" (s/2015/2014)
and removing the brand new files, They appear to reflect GHSA's preference
for double quotes over [no, single] quotes plusing adding "ghsa" and "cvss_v3" fields.
Part of #568

### CODE REVIEW

**Review by postmodern — COMMENTED** [review]
Just spotted a few inconsistencies in the formatting that I had questions about. I can merge this or if you want to `git commit --amend` the files I can wait as well.

File: gems/activerecord/CVE-2014-3483.yml [hunk 1]
```diff
@@ -3,9 +3,10 @@ gem: activerecord
 framework: rails
 cve: 2014-3483
 osvdb: 108665
+ghsa: r8fh-hq2p-7qhq
 url: https://nvd.nist.gov/vuln/detail/CVE-2014-3483
-title: "CVE-2014-3483 rubygem-activerecord: SQL injection vulnerability in 'range'\
-  \ quoting"
+title: 'CVE-2014-3483 rubygem-activerecord: SQL injection vulnerability in ''range''
+  quoting'
```
[the remark this answers is not in the document]
Comment (jasnow) [reply gems/activerecord/CVE-2014-3483.yml:9]: "We" is the "lib/github_advisory_sync.rb" script. I will manual and repush.

File: gems/awesome_spawn/CVE-2014-0156.yml [hunk 2]
```diff
@@ -1,16 +1,16 @@
...
-description: >-
-  Awesome spawn contains OS command injection vulnerability, which allows
-  execution of additional commands passed to Awesome spawn as arguments, e.g.
-  AwesomeSpawn.run('ls',:params => {'-l' => ";touch haxored"}). If untrusted
-  input was included in command arguments, attacker could use this flaw to
-  execute arbitrary command.
+description: Awesome spawn contains OS command injection vulnerability, which allows
+  execution of additional commands passed to Awesome spawn as arguments, e.g. AwesomeSpawn.run('ls',:params
+  => {'-l' => ";touch haxored"}). If untrusted input was included in command arguments,
+  attacker could use this flaw to execute arbitrary command.
```
[the remark this answers is not in the document]
Comment (jasnow) [reply gems/awesome_spawn/CVE-2014-0156.yml:11]: My post processor does the "next line" change but I did not use it on these files. Will manual edit and repush.

**Comment by jasnow** [comment]
Ready with the 2 above manually edited files. Any others?

**Comment by jasnow** [comment]
Just did
```
git add gems/activerecord/CVE-2014-3483.yml
git add  gems/awesome_spawn/CVE-2014-0156.yml
git commit -amended
git push origin master
```

**Pull request merged**